Utilities/JsonParsing.cs

```diff
@@ -42,17 +46,17 @@ public static DateTime getDateTime(string key, object val)
             }
             if (val is string str)
             {
-                if (DateTime.TryParseExact(str, "o", CultureInfo.InvariantCulture, DateTimeStyles.RoundtripKind, out var result))
+                if (DateTime.TryParseExact(str, "yyyy-MM-ddTHH:mm:ssZ", CultureInfo.InvariantCulture, DateTimeStyles.AdjustToUniversal, out var result))
                 {
                     // Journal format ("2019-09-24T02:40:34Z")
-                    return result.ToUniversalTime();
+                    return result;
                 }
-                if (DateTime.TryParseExact(str, "yyyy-MM-dd HH:mm:ss", CultureInfo.InvariantCulture, DateTimeStyles.AssumeUniversal, out result))
+                if (DateTime.TryParseExact(str, "yyyy-MM-dd HH:mm:ss", CultureInfo.InvariantCulture, DateTimeStyles.AdjustToUniversal, out result))
```

Comment:
Since EDSM time is already UTC, but has no timezone specified, then you will need `DateTimeStyles.AssumeUniversal | DateTimeStyles.AdjustToUniversal` here.